library/objective-c/EnvoyEngine.h

```diff
@@ -133,12 +139,14 @@ typedef NSDictionary<NSString *, NSArray<NSString *> *> EnvoyHeaders;
 /**
  Opens a new HTTP stream attached to this engine.
 
- @param observer Handler for stream events.
+ @param observer Handler for observing stream events.
  */
-- (EnvoyHttpStream *)openHttpStreamWithObserver:(EnvoyObserver *)observer;
+- (EnvoyStream *)startStreamWithObserver:(EnvoyObserver *)observer;
```

Comment:
My preference is to keep "Http" in here, since there are other distinct stream types, and Envoy supports them (also there's, e.g. `CFStream` vs `CFHttpStream`).

Reply:
This is to match the Java interfaces: https://github.com/lyft/envoy-mobile/pull/326/files#diff-55a9e7503aabbe4019f8a57fdf14ef86R7

Reply:
Note that the C API lacks it because it one possible route is for it to use the same interface for general stream types (and the stream handle will distinguish handling). That's not the case at this layer; it's very tied to HTTP-specific concerns and functionality.

Reply:
Also, lower level core functionality that *is* tied to HTTP is namespaced with "Http".

Reply:
I do think the Java interfaces should include "Http" as well. I had planned to update them when we started working in that layer.

Reply:
Talked offline with @buildbreaker and including HTTP here seems fine since upstream Envoy also namespaces its streams with `Http::`

Reply:
Updated